WebRoot/skins/_base/base3/splash.html

```diff
@@ -38,9 +38,6 @@ <h1><div id='ZLoginBannerImage' class='ImgLoginBanner' onclick='showCompanyUrl()
 				<div class="message">
 					<script>document.write(ZmMsg.splashScreenLoading)</script>
 				</div>
-				<div class="switch" id="splashScreenSwitchContainer">
-					<script>if(!window.isOffline) { document.write(ZmMsg.splashScreenSwitch) }</script>

```

Comment:
if ZmMsg.splashScreenSwitch is not being used anywhere else then please remove the same as well.

Reply:
Removed the "splashScreenSwitch" && "htmlClient" translations.